js/form.js

```diff
@@ -2,9 +2,19 @@ const form = document.querySelector('form');
 const emailError = document.querySelector('.form-error');
 
 form.addEventListener('submit', (e) => {
+  const name = form.name.value;
   const email = form.email.value;
+  const message = form.message.value;
   if (email !== email.toLowerCase()) {
     e.preventDefault();
     emailError.innerHTML = `<p class="error">Email should be written in lowercase letters: ${email}</p>`;
+  } else {
+    emailError.innerHTML = '';
+    const data = JSON.stringify({ name, email, message });
+    localStorage.setItem('data', data);
```

Comment:
- [ ] While loading the page the saved data should be disappeared. Make sure to fix it as well

Reply:
this was part of the requirement, did you read through to confirm

Reply:
![w-rev](https://user-images.githubusercontent.com/89055502/180473216-0fa28523-e4ef-4c3f-bbac-8f4fee9b24df.png)
this is a picture of the requirement where it specifically says when the user loads the page the input  fields are filled with pre saved data. please can you conduct a re review because i followed all of the requirement @roshan-bajgain